### PR TITLE
refactor: remove duplicate reseed command

### PR DESCRIPTION
**Pull request by emilyjablonski**
Remove the duplicate reseed command
To test:
A local reseed should seed the Detroit listings
The Travis CI tests should pass

### CODE REVIEW

File: backend/core/package.json [hunk 1]
```diff
@@ -77,7 +74,7 @@
     "express": "^4.17.1",
     "fast-xml-parser": "^4.0.0-beta.2",
     "handlebars": "^4.7.6",
-    "ioredis": "^5.2.3",
+    "ioredis": "^4.24.4",
```
Comment (ludtkemorgan) [inline backend/core/package.json:77]: Looks like this should be the 5.2.3 instead of the 4.24.4

**Review by ludtkemorgan — CHANGES_REQUESTED** [review]

File: backend/core/package.json [hunk 2]
```diff
@@ -19,13 +19,10 @@
     "db:drop": "psql -c 'DROP DATABASE IF EXISTS bloom_detroit;'",
     "db:create": "psql -c 'CREATE DATABASE bloom_detroit;'",
     "db:add-uuid-extension": "psql -d bloom_detroit -c 'CREATE EXTENSION IF NOT EXISTS \"uuid-ossp\";'",
-    "db:seed": "ts-node src/seeder/seed.ts",
```
Comment (ludtkemorgan) [inline backend/core/package.json:22]: with this change the only reference to `seed.ts` is the `test:db:setup` task. Should we also remove it from there as well?

Comment (ludtkemorgan) [reply backend/core/package.json:22]: As an addition to that, should we also remove the seed.ts file altogether?

**Comment by emilyjablonski** [comment]
Tests fail removing this file, working on getting those aligned

**Pull request closed without merging**